relayer/rpc/wormscan-client.ts

```diff
@@ -35,8 +35,35 @@ export class WormscanClient {
       );
       return { data: response.data };
     } catch (err: Error | any) {
+      return this.mapError(err);
+    }
+  }
+
+  public async getVaa(
+    chain: number,
+    emitterAddress: string,
+    sequence: bigint,
+    opts?: WormscanOptions,
+  ): Promise<WormscanResult<WormscanVaa>> {
+    try {
+      const response = await this.client.get<{ data: WormscanVaa }>(
+        `${
+          this.baseUrl
+        }api/v1/vaas/${chain}/${emitterAddress}/${sequence.toString()}`,
```

Comment:
Are we missing a backslash here `/` after `this.baseUrl`?
```suggestion
        }/api/v1/vaas/${chain}/${emitterAddress}/${sequence.toString()}`,
```

Reply:
good question.. its not needed, given `this.baseUrl` is an instance of URL, that returns the trailing `/ ` when calling toString()

Reply:
Got it, thanks 👍